oak/server/rust/oak_abi/src/lib.rs

```diff
@@ -20,6 +20,7 @@ pub use proto::oak_api::ChannelReadStatus;
 pub use proto::oak_api::OakStatus;
 
 pub mod proto;
+pub mod wasm;
```

Comment:
Personally, I'd just move the contents of `wasm/mod.rs` inline into `lib.rs`, but that would mean you'd have to do the codebase-wide update of `oak::wasm` ⇒ `oak_abi` in this PR too.  

(But it would mean the re-export that @tiziano88 mentions below wouldn't be needed.)

Reply:
Done

Reply:
I guess this is fine for now, but my point (from earlier) is that this is still part of the "dev interface" of Oak, and therefore I expect it to move it out of the server folder, and closer to the SDK folder. Not necessarily in this PR ofc.

Developers of Oak Applications should not depend on any internal or oak_runtime or oak server packages, just oak_abi and oak_sdk.